sled-agent/src/opte/mod.rs

```diff
@@ -14,30 +14,27 @@ cfg_if::cfg_if! {
 
 use ipnetwork::IpNetwork;
 use macaddr::MacAddr6;
+pub use oxide_vpc::api::BoundaryServices;
 use std::net::IpAddr;
-use std::net::Ipv6Addr;
 
-/// Location information for reaching Boundary Services, for directing
-/// inter-sled or off-rack traffic from guests.
-#[derive(Debug, Clone, Copy)]
-pub struct BoundaryServices {
-    pub ip: Ipv6Addr,
-    pub vni: Vni,
-}
+fn default_boundary_services() -> BoundaryServices {
+    use oxide_vpc::api::Ipv6Addr;
+    use oxide_vpc::api::MacAddr;
+    // TODO-completeness: Don't hardcode any of these values.
+    //
+    // Boundary Services will be started on several Sidecars during rack
+    // setup, and those addresses and VNIs will need to be propagated here.
+    // See https://github.com/oxidecomputer/omicron/issues/1382
+    let ip = Ipv6Addr::from([0xfd00, 0x99, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01]);
 
-impl Default for BoundaryServices {
-    fn default() -> Self {
-        // TODO-completeness: Don't hardcode this.
-        //
-        // Boundary Services will be started on several Sidecars during rack
-        // setup, and those addresses will need to be propagated here.
-        // See https://github.com/oxidecomputer/omicron/issues/1382
-        const BOUNDARY_SERVICES_ADDR: Ipv6Addr =
-            Ipv6Addr::new(0xfd00, 0x99, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01);
-        let boundary_services_vni = Vni::new(99_u32).unwrap();
-
-        Self { ip: BOUNDARY_SERVICES_ADDR, vni: boundary_services_vni }
-    }
+    // This MAC address is entirely irrelevant to the functionality of OPTE and
+    // the Oxide VPC. It's never used to actually forward packets. It only
+    // represents the "logical" destination of Boundary Services as a
+    // destination that OPTE as a virtual gateway forwards packets to as its
+    // next hop.
+    let mac = MacAddr::from_const([0xa8, 0x25, 0x40, 0xf9, 0x99, 0x99]);
```

Comment:
The oxide OUI is `a8:40:25`...I keep thinking we should have an oxide_mac() function or some kind of helper where we just give it the last 3 octets and the type (virt vs phys) so we never fat finger the OUI. I'm not sure where that should live (perhaps it doesn't have to live in any one place), but it's a thought I've had several times now for this very reason. Also, not saying that has to be done in this PR, just thought I'd throw out the idea.

Reply:
Well, I'll definitely _fix_ it here, thanks for catching. A function is a good idea.

Reply:
Wait, is this wrong then? https://rfd.shared.oxide.computer/rfd/0174#_mac_addresses